2023/10/22/main.go

```diff
@@ -0,0 +1,21 @@
+// Daily Joel 2023-10-22
+//
+// Switch.
+package main
+
+import (
+	"fmt"
+)
+
+func name(initial string) string {
+	switch initial {
+	case "j":
+		return "Joel"
+	default:
+		return "Unknown"
+	}
+}
+
+func main() {
+	fmt.Println(name("j"))
+}
```

Comment:
Printing function name with parameter "j", that will be assigned to the variable initial